docs/concepts.rst

```diff
@@ -32,3 +32,53 @@ Variable
 A *variable* represents a process measure of one :ref:`appliance <appliance>` of which
 timeseries data is captured and stored in the ENLYZE platform. One appliance may have
 many variables, whereas one variable is only associated with one appliance.
+
+
+.. _production_order:
+
+Production Order
```

Comment:
Alternatively, we could start describing what shop floor workers do in order to complete a production order: Produce the specified product in the specified quantity until a certain deadline. Because there might be interruptions such as weekends, high priority orders and what not, production orders might be completed within multiple intervals, which we define as *production runs*

Reply:
(Food for thought, no direct change request)

Reply:
> Alternatively, we could start describing what shop floor workers do in order to complete a production order: Produce the specified product in the specified quantity until a certain deadline. Because there might be interruptions such as weekends, high priority orders and what not, production orders might be completed within multiple intervals, which we define as production runs

But that's mostly a description of a production run, isn't it? Moreover, we'd rather have to explain *our* concept of a production order, right? And I'd say that a production order has more to do with the person planning the production than the shopfloor workers. But I'll add a sentence into that direction, see the comment below.